python/cac_tripplanner/templates/home.html

```diff
@@ -304,12 +304,30 @@
 
 {%block jspage %}
 <script type="text/javascript">
-jQuery.noConflict(); // because JotForms.
+//jQuery.noConflict(); // because JotForms.
 jQuery(document).ready(function ($) {
-    CAC.Settings.fbAppId = '{{ fb_app_id }}';
-    console.log('hi!');
+    //CAC.Settings.fbAppId = '{{ fb_app_id }}';
     //var home = new CAC.Pages.Home();
     //home.initialize();
+
+    var map = cartodb.L.map('map', {
+        center: [40.000, -75.1639],
+        zoom: 12,
+        zoomControl: false
+    });
+    var CartoDB_Positron = cartodb.L.tileLayer('http://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png', {
+    attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> &copy; <a href="http://cartodb.com/attributions">CartoDB</a>',
+    subdomains: 'abcd',
+    maxZoom: 19
+    }).addTo(map);
+
+    // set class on body
+    // FIXME: avoid having to do this for templating to work.
+    // also had to wrap contents here in a div with the same class
+    // leave out class on div or class on body, and either have no map,
+    // or map and no content overlay
```

Comment:
The top-level div in home.html: `<div class="body-home">`.

In the prototype, that div does not exist and that class exists only on the `body`. I had to add the extra div and put the class on both.